modules/core/quick/src/Form/QuickForm.php

```diff
@@ -61,7 +61,7 @@ public function getBaseFormId() {
    */
   public function getFormId() {
     $id = $this->getRouteMatch()->getParameter('id');
-    return $this->getBaseFormId() . "_$id";
+    return $this->quickFormManager->createInstance($id)->getFormId();
```

Comment:
This should use the base form ID

```suggestion
    return $this->getBaseFormId() . '_' . $this->quickFormManager->createInstance($id)->getFormId();
```

Reply:
The `QuickFormBase` class actually already prepends the form ID w/ `quick_form_` in `getFormId()`, so it already ends up as `quick_form_[id]`. The change you suggested would result in a form ID of `quick_form_quick_form_[id]`.

https://github.com/mstenta/farmOS/blob/0c8864f265581faeb98c1ee3fc83100d1284e19b/modules/core/quick/src/Plugin/QuickForm/QuickFormBase.php#L63

But maybe it would be better for `QuickFormBase` to just return the quick form ID, and not prepend `quick_form_`. Combining that with your suggested change would have the same end result (form id of `quick_form_[id]`).

I will prepare a fixup commit for consideration...

Reply:
https://github.com/mstenta/farmOS/commit/d6e23246eaf9c217d921e46e38e2eb4c179f9b90